django/contrib/admin/static/admin/js/inlines.js

```diff
@@ -97,6 +97,7 @@
                     // If a post-delete callback was provided, call it with the deleted form:
                     if (options.removed) {
                         options.removed(row);
+                        $(document).trigger('formset:removed', [row, options.prefix]);
```

Comment:
Ok, but just for clarification, these callback `options.removed`, `options.added` are always given, as they are a part of Django admin Tabular & Stacked inline. (inlines.js line:199 & 265).